db/migrate/20160607141857_change_exclude_table_settings_key.rb

```diff
@@ -0,0 +1,16 @@
+class ChangeExcludeTableSettingsKey < ActiveRecord::Migration[5.0]
+  class SettingsChange < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    serialize :value
+  end
+
+  def up
+    say_with_time("Moving exclude tables configuration") do
+      changes = SettingsChange.where(:key => "/workers/worker_base/replication_worker/replication/exclude_tables")
+      changes.each do |change|
+        change.key = "/replication/exclude_tables"
+        change.save
```

Comment:
Minor, but prefer `change.update_attributes!(:key => value)` to single line the update and save.